Documentation/contributing/release.rst

```diff
@@ -177,62 +171,83 @@ If you intent to release a new minor release, see the
        ``x.y.z`` For more information about how ReadTheDocs does versioning, you can
        read their `Versions Documentation <https://docs.readthedocs.io/en/latest/versions.html>`_.
 
-#. Build the binaries and push it to the release bucket:
-
    ::
 
-       DOMAIN=releases.cilium.io ./contrib/release/uploadrev v1.0.3
+#. Wait for DockerHub to prepare all docker images.
```

Comment:
I think the double-colon kept above this line expects a “liteal block”, but none is found. This is what CI seems to complain about. Please address :)